### DISCUSSION TITLE
Re-mapping a key that I do not know the name of (SteelSeries special key to left command on MacOS Ventura)

### DISCUSSION THREAD

**Discussion by manumbra** [pqrs-org/Karabiner-Elements#4008] [Q&A]
Hi,

I am using an older SteelSeries keyboard, but I think it apples to current models just as well. 

I want to remap the SteelSeries key situated between the Ctrl and the Alt key to the Cmd key (to have it work just like the right windows key). My problem is however, that I do not know what key I should remap. I wonder if there could be a feature, that doesn`t remap keys from a list, but by pressing them. 

Does anyone know how to address this? 
For now I have mapped the left Ctrl to the left Cmd, and I come by just fine with just the right Ctrl. But I assume this is a good usecase for Karabiner.

**Comment by tekezo** [comment]
You can check this in EventViewer.
(EventViewer displays the keys after modification, so if you need to know the original key names, try using an empty profile or similar.)
https://karabiner-elements.pqrs.org/docs/manual/operation/eventviewer/
